.babelrc

```diff
@@ -31,5 +31,15 @@
         ]
       ]
     }
-  }
+  },
+  "plugins": [
```

Comment:
This is optional. It obviates the requirement for an `id` prop in message descriptors. See https://formatjs.io/docs/getting-started/message-extraction